third_party/gfootball_engine/src/data/teamdata.cpp

```diff
@@ -198,7 +198,11 @@ TeamData::TeamData(int teamDatabaseID, const std::vector<FormationEntry> &f) {
 
   while (changed && iterations < maxIterations) {
     DO_VALIDATION;
+#ifdef WIN32
+    std::vector<Vector3> offset(player_count);
+#else
     Vector3 offset[player_count];
```

Comment:
Hi @vi3itor, do you consider remove Variable Length Arrays in next commit? I think it will be better use std::vector for both MSVC and Clang.